src/store/modules/swap/ethBancor.ts

```diff
@@ -3765,10 +3792,23 @@ export class EthBancorModule
   }
 
   @action async getMaxStakes({ poolId }: { poolId: string }) {
-    const [balances, poolTokenBalance, isHighTierPool] = await Promise.all([
+    const contract = await buildLiquidityProtectionSettingsContract(
+      this.contracts.LiquidityProtectionSettings,
+      w3
+    );
+
+    const [
+      balances,
+      poolTokenBalance,
+      isHighTierPool,
```

Comment:
Do we no longer need this information? ```poolTokenBalance``` and ```isHighTierPool```

Reply:
thanks for the hint, removed